pkg/harvester/list/devices.harvesterhci.io.usbdevice.vue

```diff
@@ -0,0 +1,88 @@
+<script>
+import { HCI } from '../types';
+import { STATE, SIMPLE_NAME } from '@shell/config/table-headers';
+import { allHash } from '@shell/utils/promise';
+import Banner from '@components/Banner/Banner.vue';
+import Loading from '@shell/components/Loading';
+import MessageLink from '@shell/components/MessageLink';
+import { ADD_ONS } from '../config/harvester-map';
+import DeviceList from '../edit/kubevirt.io.virtualmachine/VirtualMachineUSBDevices/DeviceList';
+
+export default {
+  name: 'ListUsbDevicePage',
+
+  components: {
+    Banner,
+    DeviceList,
+    Loading,
+    MessageLink,
+  },
+
+  async fetch() {
+    const inStore = this.$store.getters['currentProduct'].inStore;
+
+    this.schema = this.$store.getters[`${ inStore }/schemaFor`](HCI.USB_DEVICE);
+    this.hasAddonSchema = this.$store.getters[`${ inStore }/schemaFor`](HCI.ADD_ONS);
+
+    if (this.hasSchema) {
+      try {
+        const hash = await allHash({
+          usbDevices: this.$store.dispatch(`${ inStore }/findAll`, { type: HCI.USB_DEVICE }),
+          addons:     this.$store.dispatch(`${ inStore }/findAll`, { type: HCI.ADD_ONS }),
+        });
+
+        this.hasPCIAddon = hash.addons.find(addon => addon.name === ADD_ONS.PCI_DEVICE_CONTROLLER)?.spec?.enabled === true;
+      } catch (e) {}
+    }
+  },
+
+  data() {
+    return {
+      hasAddonSchema: false,
+      hasPCIAddon:    false,
+      schema:         null,
+      toUSBAddon:     `${ HCI.ADD_ONS }/harvester-system/${ ADD_ONS.USB_DEVICE_CONTROLLER }?mode=edit`,
+      headers:        [
+        { ...STATE },
+        SIMPLE_NAME,
+      ],
+    };
+  },
+
+  computed: {
+    hasSchema() {
+      return !!this.schema;
+    },
+
+    devices() {
```

Comment:
Can we add devices in data and filled when fetch?
Do we need another devices() in computed ?

```
this.devices = hash.usbDevices;
.....
devices : []

```

Reply:
It has to be a computed property to be update when enabling or disabling the devices.